marlowe/cli-doc/simple-test.sh

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env bash
+
+
+# Simple end-to-end test of `marlowe-cli`.
+#
+#
+# The environment variable CARDANO_NODE_SOCKET_PATH must be set.
+#
+# The following tools must be on the path:
+#   marlowe-cli
+#   cardano-cli
+#   sed
+#   jq
+#   timeout
+#
+# This script exits with an error value if the end-to-end test fails.
+
+
+set -ex
+
+
+# Select the network.
+
+NETWORK=testnet
+MAGIC_FLAG=--testnet-magic
+MAGIC_NUM=1097911063
+
+if [ -z "$CARDANO_NODE_SOCKET_PATH" ]
+then
+  export CARDANO_NODE_SOCKET_PATH=$PWD/$NETWORK.socket
+fi
+
+
+# Select the wallet.
+
+PAYMENT_SKEY=payment.skey
+PAYMENT_VKEY=payment.vkey
```

Comment:
There should be instructions stating that these two files are required for the test.

Reply:
@bwbush the [cardano docs](https://developers.cardano.org/docs/get-started/running-cardano/#running-the-node) use the following convention: 
```suggestion
PAYMENT_VKEY=path/to/payment.vkey
```

Reply:
Fixed in 70b73b0f9cee949c16716ccd93635745080999b5.